### PR TITLE
Add parameter for internal/external emitter-package.json

### PR DESCRIPTION
**Pull request by hallipr**

### CODE REVIEW

File: eng/pipelines/new-emitter-package-files.yml [hunk 1]
```diff
@@ -9,16 +9,22 @@ parameters:
...
+- name: OutputDirectory
+  type: string
+- name: UseInternalFeed
+  type: bool
 
 steps:
 - pwsh: |
-    $feedUrl = "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-js-test-autorest@local/npm/registry"
-
+    $useInternalFeed = '${{ parameters.UseInternalFeed }}' -eq 'true
```
Comment (m-nash) [inline eng/pipelines/new-emitter-package-files.yml:19]: is this missing a `'` in true?

Comment (hallipr) [reply eng/pipelines/new-emitter-package-files.yml:19]: yep

Comment (hallipr) [reply eng/pipelines/new-emitter-package-files.yml]: fixed

**Review by m-nash — APPROVED** [review]

**Comment by hallipr** [comment]
/check-enforcer override

The changes will not be tested by PR pipeline code paths

**Pull request merged**